Source/WebCore/page/DOMTimer.cpp

```diff
@@ -93,7 +93,7 @@ class DOMTimerFireState {
     static DOMTimerFireState* current;
 
 private:
-    ScriptExecutionContext& m_context;
+    Ref<ScriptExecutionContext> m_context;
```

Comment:
DOMFireState is a stack object so this seems safe.

Reply:
That's correct, but nothing stops this from being created on the heap. I think the intent is to convert member variables that can support smart pointers to adopting them.